src/unistd.rs

```diff
@@ -701,6 +701,28 @@ pub fn execvp(filename: &CString, args: &[CString]) -> Result<Void> {
     Err(Error::Sys(Errno::last()))
 }
 
+/// Replace the current process image with a new one and replicate shell `PATH`
+/// searching behavior (see
+/// [execvpe(3)](http://man7.org/linux/man-pages/man3/exec.3.html)).
+///
+/// This is a combination of `::nix::unistd::execvp` and `::nix::unistd::execve`
```

Comment:
This wording suggests that nix itself is implementing the combination of features, when it's actually the OS.  I suggest this wording: "This functions like a combination of `execvp(2)` and `execve(2)` ..."

Reply:
What about this part?

Reply:
Sorry, didn't read close enough. I've fixed it now.